archeryutils/handicaps/handicap_tables.py

```diff
@@ -267,15 +267,13 @@ def _check_print_table_inputs(
             If no rounds are provided for the handicap table
 
         """
-        if not isinstance(hcs_in, np.ndarray):
-            if isinstance(hcs_in, list):
-                hcs_in = np.array(hcs_in)
-            elif isinstance(hcs_in, (float, int)):
-                hcs_in = np.array([hcs_in])
-            else:
-                msg = "Expected float or ndarray for hcs."
-                raise TypeError(msg)
-        hcs = hcs_in.astype(float)
+        hcs_in = np.asarray(hcs_in)  # Explicitly cast to array for calculations
+
+        try:
+            hcs = hcs_in.astype(np.float64)
+        except ValueError as exc:
+            msg = "Cannot convert supplied handicaps to float for HandicapTable."
+            raise TypeError(msg) from exc
```

Comment:
```suggestion
        try:
            hcs = np.asarray(hcs_in, dtype=float)
        except ValueError as exc:
            msg = "Cannot convert supplied handicaps to float for HandicapTable."
            raise TypeError(msg) from exc
```

Reply:
done in https://github.com/jatkinson1000/archeryutils/pull/133/commits/954a89bbba53c483db18b5cabd1eb91fac2ad010 but explicitly using `np.float64` to match the rest of the code.